package.json

```diff
@@ -40,24 +40,24 @@
   },
   "peerDependencies": {
     "react": ">=16.8.6",
-    "react-native": ">=0.59.8"
+    "react-native": ">=0.65.0"
   },
   "devDependencies": {
-    "@types/jest": "^27.4.0",
+    "@types/jest": "^27.4.1",
     "@types/react": "^17.0.39",
-    "@types/react-native": "^0.66.15",
-    "@typescript-eslint/eslint-plugin": "^5.10.2",
-    "@typescript-eslint/parser": "^5.10.2",
-    "eslint": "^8.8.0",
-    "eslint-plugin-jest": "^26.1.0",
+    "@types/react-native": "^0.67.1",
```

Comment:
I'm not an expert of typescript so probably I miss something. I'm a bit confused by this version number.
If we support RN 0.65+ I would expect the types <0.65 (here it's ^0.67).
I've tried to find a place where I can see correspondence between types and related libs but I haven't found anything.